### PR TITLE
PR: modification of edit article page for refresh problem.

### PR DESCRIPTION
**Pull request by creative2113**
I have fixed the refresh problem of edit article page.
please check this.
thank you.

### CODE REVIEW

**Review by shinevue — APPROVED** [review]
Great, go on

File: client/src/components/Input/TextAreaInput.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
-import { useState } from "react";
 
-function TextAreaInput({ labelTitle, labelStyle, name, containerStyle, placeholder, className, onChange }) {
+function TextAreaInput({ labelTitle, labelStyle, name, containerStyle, placeholder, className, onChange, value }) {
     return (
         <div className={`form-control w-full flex sm:flex-row flex-col ${containerStyle}`}>
             <label className="label w-1/5">
```
Comment (shinevue) [inline client/src/components/Input/TextAreaInput.js:5]: add label style to label element not only span.
And reorder props for easy-show

**Review by shinevue — CHANGES_REQUESTED** [review]

File: client/src/components/Input/TextAreaInput.js (hunk 1 above)
Comment (creative2113) [reply client/src/components/Input/TextAreaInput.js:5]: Hello.
I will fix that.
let me give just a minute.

**Review by shinevue — APPROVED** [review]
Nice, go ahead!

**Pull request merged**